sdk/compute/Azure.ResourceManager.Compute/tests/Helpers/ResourceDataHelper.cs

```diff
@@ -185,6 +186,31 @@ public static GalleryImageIdentifier GetGalleryImageIdentifier(string publisher,
         }
         #endregion
 
+        #region GalleryImageVersion
+        public static void AssertGalleryImageVersion(GalleryImageVersionData imageversion1, GalleryImageVersionData imageversion2)
```

Comment:
Please change the variable name to camel case
```suggestion
        public static void AssertGalleryImageVersion(GalleryImageVersionData imageVersion1, GalleryImageVersionData imageVersion2)
```